does not work with non-RSA ssh keys

On my workstation, I use `ed25519` SSH keys. Something in `make instance.setup` is ignoring the `SSH_KEY` set in `.configs.anders` (or as a variable in the shell) and defaulting to `$HOME/.ssh/id_rsa`, which in my case doesn't exist.

eg,

```
grep SSH_KEY .configs.anders                                                                             
SSH_KEY="$(HOME)/.ssh/id_ed25519"
$ make instance.setup                                                      
Flush pip packages...
make[1]: Entering directory '/home/anders/work/sultan'
Installing project requirements...
make[2]: Entering directory '/home/anders/work/sultan'
Updating your inventory credentials...    (dynamic-inventory/gce.ini)
/home/anders/.ssh/id_rsa: No such file or directory
make[2]: *** [targets/local.mk:47: local.inventory.config] Error 1
make[2]: Leaving directory '/home/anders/work/sultan'
make[1]: *** [Makefile:38: ve/bin/ansible-playbook] Error 2
make[1]: Leaving directory '/home/anders/work/sultan'
make: *** [Makefile:68: clean] Error 2
```

Same error even if I do `SSH_KEY=/home/anders/id_ed25519 make instance.setup`

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-include .configs*
+include $(sort $(wildcard .configs*))
 export $(shell sed 's/=.*//' .configs*)
 
 SHELL := /bin/bash

